src/scss/grommet-core/_objects.heading.scss

```diff
@@ -0,0 +1,17 @@
+.heading {
+  margin-bottom: halve($inuit-base-spacing-unit);
+
+  &--large span {
+    font-size: 125%;
+    line-height: 60px;
```

Comment:
I think the line-height will need to be adjusted relatively as well. We won't a 60px line-height for a small h5, for example. I'd set it to 1.2 for now and we can adjust when we tune the sizes.